ComplexTrStructure/ComplexTrStructure/ComplexTr.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ComplexTrStructure
+{
+    public class ComplexTr
+    {
+        double abs;
+
+        public double Abs
+        {
+            get => abs;
+
+            set
+            {
+                if (value < 0)
+                    throw new ArgumentException("Значение должно быть неотрицательным");
+                abs = value;
+            }
+        }
+        double arg;
+        public double Arg
+        {
+            get => arg;
+            set => arg = value;
+        }
+        public double Re
+        { 
+            get => Math.Round(Abs* Math.Cos(arg), 13);
+        }
+
+        public double Im
+        {
+            get => Math.Round(Abs * Math.Sin(arg), 13);
+        }
+
+        public ComplexTr(double abs, double arg)
+        {
+            this.Abs = abs;  
+            this.Arg = arg;  
```

Comment:
В структурах не всё так, как в классах. В частности, нельзя использовать экземпляр структуры (в частности, обращаться к его свойству) пока не заполнены все поля структуры.
```suggestion
            this.abs = abs;  
            this.arg = arg;  
```